VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -35,21 +42,33 @@ protected void onCreate(Bundle savedInstanceState) {
 
         webView = (WebView) findViewById(R.id.webview);
         webView.loadUrl(environment.getConfig().getCourseDiscoveryConfig().getCourseSearchUrl());
+
+        DrawerLayout mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+        mDrawerToggle = new ActionBarDrawerToggle(this, mDrawerLayout,
+                R.string.label_open_drawer, R.string.label_close_drawer) {
+            @Override
+            public void onDrawerSlide(View drawerView, float slideOffset) {
+                super.onDrawerSlide(drawerView, slideOffset);
+                invalidateOptionsMenu();
+            }
+        };
+        mDrawerLayout.setDrawerListener(mDrawerToggle);
```

Comment:
This should be `addDrawerListener()`, cause `BaseFragmentActivity` already has a listener on it, which would be supplemented. The set method has been deprecated in any case.